Procfile

```diff
@@ -1,2 +1,3 @@
 web: bundle exec puma -C config/puma.rb
-
+worker: bundle exec rake jobs:work
+release: bundle exec rake db:migrate && bundle exec rake db:seed
```

Comment:
Wel raar, vsv heeft namelijk ook een app.json waarin deze stap al staat bij een postdeploy. Maar blijkbaar werkt die hier niet (of doen we de migrates een seeds van hgi ook handmatig?). Iig volgens mij werkt het nu wel.